### PR TITLE
fixe snyk Vulnerabilities #553

### PR DESCRIPTION
**Pull request by kslazykv**
[snyk](https://app.snyk.io/org/procosys/project/a4057c4e-43f7-4af0-a5cf-c68c1c6ebeef )

[#553](https://github.com/orgs/equinor/projects/721/views/28?filterQuery=&pane=issue&itemId=82699658&issue=equinor%7Ccc-toolbox%7C553)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,12 +54,15 @@
     "serialize-javascript": "6.0.2",
     "socket.io": "4.6.2",
     "socket.io-parser": "4.2.3",
-    "styled-components": "^5.3.6",
+    "styled-components": "5.3.7",
+    "tough-cookie": "4.1.3",
     "tweetnacl": "1.0.3",
-    "uglify-js": "2.4.24",
+    "ua-parser-js": "1.0.33",
+    "uglify-js": "3.14.3",
```
Comment (larskay) [inline package.json:61]: You sure there are no breaking changes here?

File: package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@adobe/css-tools": "4.3.2",
     "@azure/msal-browser": "2.32.2",
-    "@babel/traverse": "7.23.2",
+    "@babel/traverse": "8.0.0-alpha.4",
```
Comment (larskay) [inline package.json:9]: And here as well

Comment (larskay) [reply package.json:9]: Not sure if we should use alpha versions

File: package.json (hunk 1 above)
Comment (kslazykv) [reply package.json:61]: I have checked and did not find any issues in the application. After updating the packages, I will submit the app for UAT testing.

**Review by larskay — APPROVED** [review]

**Pull request merged**